src/components/create-class/create-class.tsx

```diff
@@ -0,0 +1,27 @@
+// ~/components/create-class/create-class.tsx
+'use client';
+import { useRouter } from 'next/navigation';
+import { Button } from '../ui/button'; // Asegúrate de que esta ruta sea correcta
+
+interface CreateClassButtonProps {
+  className?: string; // Aceptar className como prop
+}
+
+export const CreateClassButton: React.FC<CreateClassButtonProps> = ({
+  className,
+}) => {
+  const router = useRouter();
+
+  const handleClick = () => {
+    router.push('/courses/create');
+  };
+
+  return (
+    <Button
```

Comment:
Use the Next `<Link />` component to redirect to create a course and the `asChild` prop to allow the link to have the styles of the button